mriqc/bin/mriqc_run.py

```diff
@@ -252,14 +269,30 @@ def main():
     # Process data types
     modalities = opts.modalities
 
-    dataset = collect_bids_data(
-        settings['bids_dir'],
-        modalities=modalities,
-        participant_label=opts.participant_label,
-        session=opts.session_id,
-        run=opts.run_id,
-        task=opts.task_id,
-    )
+    if opts.T1w_file:
+        dataset=dict()
```

Comment:
`dataset = {'T1w': [settings['bids_dir']], 'bold': [], 'T2w': []}`

Could you check if it breaks when you don't even set the `'bold'` and `'T2w'` keys?